docs/examples/pytorch/resnet50/main.py

```diff
@@ -106,7 +109,10 @@ def define_graph(self):
         rng = self.coin()
         self.jpegs, self.labels = self.input(name="Reader")
         images = self.decode(self.jpegs)
-        images = self.rrc(images)
+        if self.dali_cpu:
```

Comment:
Maybe it is better to name it `self.res` in both cases and get rid of that if here.

Reply:
Will do